<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>public_user</class>
 <widget class="QWidget" name="public_user">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>後台人員系統</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
	font-size:24px;
    font-family: Arial;
    color: white;
    background-color: #007BFF;
    border-radius: 8px;
    border: 1px solid #0056b3;
    /* 設定按鈕的最小尺寸 */
    min-width: 100px;
    min-height: 50px; /* 可選，設置最小高度 */
}
QPushButton:hover {
    background-color: #0056b3;
}
1</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Policy::Minimum</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="refresh_button_2">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>重新整理</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="add_button_2">
             <property name="text">
              <string>添加</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Policy::Minimum</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QTableWidget" name="tableWidget_2"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
